src/main/scala/inox/solvers/smtlib/SMTLIBTarget.scala

```diff
@@ -428,6 +428,14 @@ trait SMTLIBTarget extends SMTLIBParser with Interruptible with ADTManagers {
       case BVAShiftRight(a, b)       => FixedSizeBitVectors.AShiftRight(toSMT(a), toSMT(b))
       case BVLShiftRight(a, b)       => FixedSizeBitVectors.LShiftRight(toSMT(a), toSMT(b))
 
+      case c @ BVWideningCast(e, _)  =>
+        val Some((from, to)) = c.cast // FIXME can we assume it's well typed here?
```

Comment:
Yes, you can assume they're well typed. Inox should have complained earlier otherwise.